evalml/pipelines/component_graph.py

```diff
@@ -24,6 +24,9 @@
     infer_feature_types,
 )
 
+# from tkinter import W
+
+
```

Comment:
Can you remove these lines completely?

Reply:
Agreed, let's try to refrain from committing any unused, commented code.